### ISSUE TITLE
Can't unreact 

### ISSUE THREAD

**Issue by lukasdeveloper** [FriendsOfFlarum/reactions#27]
Hello,

i have the odd issue of not being able to unreact on a post. I have debug mode enabled and get the following error:

```{
  "errors": [
    {
      "status": "422",
      "code": "validation_error",
      "detail": "You can't use this reaction right now, please refresh the page",
      "source": {
        "pointer": "/data/attributes/message"
      }
    }
  ]
}
```

I hope that helps to fix the issue.

Kind Regards.

**Comment by datitisev** [comment]
Hm, for some reason I can't reproduce this issue locally.

Can you install the latest commit with `compose require fof/reactions dev-master` and see if it fixes the issue?

**Comment by lukasdeveloper** [comment]
Hello, i just updated to dev-master and it kinda works again. I can unreact but still get an error but after reloading the website the react is gone.
Here the ERROR:
```{
  "errors": [
    {
      "status": "500",
      "code": "unknown",
      "detail": "TypeError: Argument 3 passed to FoF\\Reactions\\Listener\\SaveReactionsToDatabase::push() must be an instance of FoF\\Reactions\\Reaction, null given, called in ```

And thats the error in network when i unreact. 
![image](https://user-images.githubusercontent.com/31061174/82758151-290cd580-9de5-11ea-91ed-c5dc23b1f30a.png)

**Comment by datitisev** [comment]
I need the HTTP request data of that request that you showed. 

Something like this
![image](https://user-images.githubusercontent.com/6401250/82760341-da4d4480-9dc0-11ea-8838-46afebb309cd.png)

I pushed another commit. Require dev-master again and see if it fixes the issue. Version downloaded should say `dev-c29cdb0` when it updates it.

**Comment by lukasdeveloper** [comment]
Hello,
yes your latest version works :)

**Comment by datitisev** [comment]
Ok, thank you. I will release a hotfix later today.

**Comment by datitisev** [comment]
Released version `0.3.2`.
